.github/workflows/local_tests.yml

```diff
@@ -48,13 +48,23 @@ jobs:
             mandrel-version: '22.3.1.0-Final'
             java-version: '11'
 
-          - quarkus-version: '2.16.0.Final'
+          - quarkus-version: '2.16.1.Final'
             mandrel-version: '21.3.5.1-Final'
             java-version: '11'
-          - quarkus-version: '2.16.0.Final'
+          - quarkus-version: '2.16.1.Final'
             mandrel-version: '21.3.5.1-Final'
             java-version: '17'
```

Comment:
I think that you can just do:

```
exclude:
          - quarkus-version: '2.16.1.Final'
            mandrel-version: '21.3.5.1-Final'
```

and it will exclude both Java 11 and 17.

See https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategymatrixexclude